src/api/agents/adapters.ts

```diff
@@ -43,9 +43,10 @@ function isInTest(config?: AgentConfig) {
   if (config && Object.keys(config).length === 0) {
     return true;
   }
-
+  
+  const phoneNumber = config?.integration_settings?.order_status_restriction?.phone_numbers
   const hasPhoneNumber = config?.integration_settings?.order_status_restriction?.phone_numbers
```

Comment:
Shouldn't `phoneNumber` be used here?
```suggestion
  const phoneNumber = config?.integration_settings?.order_status_restriction?.phone_numbers
  const hasPhoneNumber = phoneNumber
```